src/inject_key_bin.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+
+#
+# Use this script to inject your own private key and authentication counter
+# into U2F binary. Might be useful if you want keys to survive firmware updates.
+#
+# Example:
+#
+# Generate EC private key with openssl:
+# > openssl ecparam -name prime256v1 -genkey -noout -outform der > key.der
+#
+# Inject generated key into u2f.bin and set auth counter to 100:
+# > python3 inject_key_bin.py --key key.der --ctr 100
+#
+# key will not be modified if --key parameter is not present
+# counter will not be modified if --ctr parameter is not present
+
+from __future__ import print_function
+from asn1crypto.keys import ECPrivateKey
+import hashlib
+import argparse
+import sys
+import struct
+import os
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--bin", default="build/u2f.bin",
```

Comment:
Can we combine inject_key and inject_key_bin into one? Or even replace inject_key with the inject_key_bin...

Reply:
I think once inject_key_bin is proven to work, it can replace inject_key.